wellcomecollection/wellcomecollection.org
# Bump node version for prismic-model to 20

## What does this change?

- Bumps Node version to 20 for the prismic-model repo
- Removes unused Babel package and plugins

## How to test

- `yarn`
- run any of the scripts in `package.json` (e.g. `yarn sliceAnalysis`) and confirm they still work

## How can we measure success?

N/A

## Have we considered potential risks?

Can't think of any
